### PR TITLE
Several entries added

### PR DESCRIPTION
**Pull request by swallstrom**
Facility, code, and web service entries related to submm observations and analysis with the JCMT and APEX telescopes added

### CODE REVIEW

File: entries/codes/starlink.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Starlink
+category: codes
+tags: 
+text: The Starlink software (Currie et al. 2014) is currently supported by the East Asian Observatory.
+latex: The Starlink software \citep{Currie_2014} is currently supported by the East Asian Observatory.
+url: https://starlink.eao.hawaii.edu/starlink
+dependencies: 
+bibtex: @INPROCEEDINGS{Currie_2014,
```
Comment (hamogu) [inline entries/codes/starlink.yaml]:
```suggestion
bibtex: >
  @INPROCEEDINGS{Currie_2014,
```

**Review by hamogu — COMMENTED** [review]
Thanks for your contribution. Unfortunately, at least one file has a YMAL error in it. I know it's a little obscure format, so I'll try top fix it for you.

File: entries/codes/starlink.yaml (hunk 1 above)
Comment (swallstrom) [reply entries/codes/starlink.yaml]: Thanks for the fix!

File: entries/codes/starlink.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: Starlink
+category: codes
+tags: 
+text: The Starlink software (Currie et al. 2014) is currently supported by the East Asian Observatory.
+latex: The Starlink software \citep{Currie_2014} is currently supported by the East Asian Observatory.
+url: https://starlink.eao.hawaii.edu/starlink
+dependencies: 
+bibtex: >
+@INPROCEEDINGS{Currie_2014,
```
Comment (hamogu) [inline entries/codes/starlink.yaml]:
```suggestion
  @INPROCEEDINGS{Currie_2014,
```

File: entries/codes/starlink.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+       series = {Astronomical Society of the Pacific Conference Series},
+       volume = {485},
+        month = may,
+        pages = {391},
+       adsurl = {https://ui.adsabs.harvard.edu/abs/2014ASPC..485..391C},
+      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+}
+
+
+
```
Comment (hamogu) [inline entries/codes/starlink.yaml]:
```suggestion
  }
```
Stupid white sapce requirements ;-)

**Comment by hamogu** [comment]
Seems we squashed all those white space errors and sorry that I needed more than one commit for that - I don't do that that often either ;-)
Thanks a lot for contributing!

**Pull request merged**